Source/DataModels/Camera/Camera.cpp

```diff
@@ -47,7 +46,7 @@ Camera::Camera(Camera& camera)
 	lastMouseX(camera.lastMouseX),
 	lastMouseY(camera.lastMouseY),
 	mouseState(camera.mouseState),
-	frustum(std::move(camera.frustum))
+	frustum(new Frustum(*camera.frustum))
```

Comment:
Instead of changing the frustum to a raw pointer (which you aren't deleting, if I didn't miss it), you could do something like
```suggestion
	frustum(std::unique_ptr<Frustum>(new Frustum(*camera.frustum)))
```